src/paperless/settings.py

```diff
@@ -589,6 +589,10 @@ def default_threads_per_worker(task_workers) -> int:
 # Specify the filename format for out files
 PAPERLESS_FILENAME_FORMAT = os.getenv("PAPERLESS_FILENAME_FORMAT")
 
+# If this is enabled, variables in filename format with resolve to empty-string instead of none.
```

Comment:
Typo? "...variables in filename format **will** resolve to..."?

Reply:
Updated